src/Stripe.net.Tests/Stripe.net.Tests.csproj

```diff
@@ -20,7 +21,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.*" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net46' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">
```

Comment:
Good catch. It was multiple frameworks before and two different sections (the other removed) 👍